MrTheGood/projectb_android
# Release 2.1

Changes:
 * Update theme preview images
 * Add privacy policy button to follow the Google Play store rules.
 * Update dependencies
 * Released 2.1 to Google Play and Amazon App Store

How to test: 
 1. Check the theme preview images in Settings
 2. Click the Privacy Policy button in Settings
 3. Or just accept. I'll send updated links to the play store ASAP.
